actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkUploadBackgroundJobActor.java

```diff
@@ -125,7 +125,8 @@ private void processUser(BulkUploadProcessTask task, String organisationId, Stri
       try {
         String roles = (String) userMap.get(JsonKey.ROLES);
         if (roles != null) {
-          userMap.put(JsonKey.ROLES, Arrays.asList(roles.split("\\\\s*,\\\\s*")));
+          String[] roleList = roles.split(",");
```

Comment:
in Roles empty space need to be trim as well. in previous code u are doing.

userMap.put(JsonKey.ROLES, Arrays.asList(roles.split("\\\\s*,\\\\s*")));